### ISSUE TITLE
Error: Unable to erase space

### ISSUE THREAD

**Issue by mishaone** [openshiftio/openshift.io#651]
I'm using the "Reset Environment" option to clear out my account but one space consistently cannot be erased. See screenshot for error. I'm not sure if there is something we're supposed to do (maybe in OSO to "free" the resources) before using the reset environment option, but this would be good to document if it's a feature, not a bug. 
![unable_to_erase](https://user-images.githubusercontent.com/926572/29645195-3a6a133a-88bf-11e7-81fe-ee49b5d6fe5c.png)

**Comment by joshuawilson** [comment]
@aslakknutsen @dgutride any ideas?

**Comment by dgutride** [comment]
@joshuawilson  - nothing on the front end should prevent this - maybe @aslakknutsen could see if there is something happening on the backend?

**Comment by mishaone** [comment]
Quick update (or maybe I just noticed this now) but the erase environment option can't erase the space, but it shows a success message (see screenshot) despite this, which doesn't seem ideal.

![success](https://user-images.githubusercontent.com/926572/29951771-527c8288-8f08-11e7-8523-d2011c686cc1.png)

**Comment by alexeykazakov** [comment]
For some reason there is no resource which represents this particular space. It may happen with some old spaces if Keycloak failed to create a resource when the space was being created.
That was fixed by https://github.com/fabric8-services/fabric8-wit/pull/1589 so, now new spaces will not be created if there is any problem with creating a resource for that space.  But previously the space created anyway.
I've created a PR to ignore such errors and force space removing even if there is no space resource - 
https://github.com/fabric8-services/fabric8-wit/pull/1604
As soon as this PR merged and deployed to Prod this space will become removable.
